platform/lambda/lambda.go

```diff
@@ -654,6 +747,10 @@ func (p *Platform) removeProxy() error {
 	return nil
 }
 
+func (p *Platform) getS3Key(stage string) string {
+	return fmt.Sprint(p.config.Name, "-", stage, ".zip")
```

Comment:
I think we'll need some randomness here, otherwise concurrent deploys from teammates will start overriding each other.